packages/devextreme/js/__internal/scheduler/utils/data_accessor/AppointmentDataAccessor.test.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, expect, it } from '@jest/globals';
+import { AppointmentDataAccessor } from '@ts/scheduler/utils';
+
+describe('AppointmentDataAccessor', () => {
+  it('should get custom property', () => {
+    const dataAccessor = new AppointmentDataAccessor({ propExpr: 'prop' } as any, false);
+
+    expect(dataAccessor.get('prop', { prop: 1 })).toBe(1);
+  });
+  it('should set custom property', () => {
```

Comment:
Let's add empty line between test cases
It makes them better redable 

```suggestion
  });
  
  it('should set custom property', () => {
```